src/Generator/OutputGeneratorTrait.php

```diff
@@ -60,7 +60,7 @@ protected function createResponseDenormalizationStatement($status, $schema, Cont
                         new Expr\PropertyFetch(new Expr\Variable('this'), 'serializer'),
                         'deserialize',
                         [
-                            new Arg(new Expr\MethodCall(new Expr\MethodCall(new Expr\Variable('response'), 'getBody'), 'getContents')),
+                            new Arg(new Expr\Cast\String_(new Expr\MethodCall(new Expr\Variable('response'), 'getBody'))),
```

Comment:
Sure PSR2 force about using a cast instead of __toString ? And no it cannot be null, it can only be an empty stream (but still respecting the StreamInterface). Anyway don't mind :)